browser/todo-list.md

```diff
@@ -3,291 +3,248 @@
 
-![](images/make-a-list.png)
+### Sign out and see your public tasks
 
-Each list is immediately stored in the Gaia Hub linked to your Blockstack ID.
-For more information about the Gaia hub, [see the overview in this documentation]({{ site.baseurl }}/storage/overview.html#). Now that you have seen the application in action, dig into how it works.
+Select "Sign out" to deauthenticate the app with your Blockstack account.
 
+This triggers an event, which [under the hood](https://github.com/blockstack/blockstack-todos/blob/master/src/components/Header.jsx#L47) calls the [`signUserOut` method](https://blockstack.github.io/blockstack.js/classes/usersession.html#signuserout) of the `UserSession` object.
 
-## Task 7: Implement storage
+Now, visit the URL that was provided to you when you made your tasks public. This url is of the format `/todos/:username`, so if your username is `jane_doe.id.blockstack`, the URL would be [`localhost:3000/todos/jane_doe.id.blockstack`](http://localhost:3000/todos/jane_doe.id.blockstack).
 
-Go to the underlying `blockstack-todo` code you cloned or downloaded. The application interactions with your Gaia Hub originate in the `src/components/Profile.js` file. First, examine where the changes to the Todos are processed in the `Profile.js` file.
+When you visit this page, the `TodoList.jsx` component detects that there is a username in the URL. When there is a username, it calls `fetchTasks`, this time providing the `username` argument. This `username` option is then passed to `getFile`, which will lookup where that user's tasks are stored.
 
-The code needs to read the Todo items from the storage with the [`getFile()`](https://blockstack.github.io/blockstack.js/#getfile) method which returns a promise:
+### Sign back in
 
-```js
-  loadTasks() {
-    const options = { decrypt: true };
-    this.props.userSession.getFile(TASKS_FILENAME, options)
-    .then((content) => {
-      if(content) {
-        const tasks = JSON.parse(content);
-        this.setState({tasks});
-      } 
-    })
-  }
-```
-The `todos` data is retrieved from the promise. By default, the `getFile()` decrypts data for you. For more information on the available options, see the <a href="https://blockstack.github.io/blockstack.js/interfaces/getfileoptions.html" taraget="_blank">the blockstack.js</a> library for details on the `GetFileOptions` interface.
+At this point, you will be logged out from the app but not you'll still have an active session with the Blockstack app itself on [app.blockstack.org](https://app.blockstack.org). Navigate to app.blockstack.org and select "Sign out" there if you want to deauthenticate the Blockstack app as well.
 
-During the creation of a  `todos`, a JSON object is passed in and the [`putFile()`](https://blockstack.github.io/blockstack.js/#putfile) method to store it in a Gaia Hub. By default, `putFile()` encrypts data when it stores it.
+Once signed out, select "Sign in" to sign back in with your *Secret Key*.
```

Comment:
I'd make clear that "Sign in" is on the homepage of the sample app, not the "Blockstack app"